apps/server/src/connect/userEndpoints.test.ts

```diff
@@ -75,7 +75,7 @@ describe('userEndpoints', () => {
 
       await userDeleteHandler(req, res)
 
-      expect(res.send).toHaveBeenCalledWith('')
+      expect(res.send).toHaveBeenCalledWith('User delete failed')
```

Comment:
For some reason this is different than before. I assume it is because of how errors are handled in the actual mx platform sdk.

Reply:
What would be a better `it` description?

Reply:
It's not using any specific stuff related to a provider api failing. I'd just call it "responds with a failure if mx deletion api fails"

Reply:
Done